2021-01-25 17:31:47,530 - INFO - domain_parser.parse() is running
2021-01-25 17:31:48,035 - INFO - domain_parser.parse() is running
2021-01-25 17:31:48,383 - INFO - Domain Parse process is done 25797 unique URLs are parsed
2021-01-25 17:31:48,549 - INFO - parse Dosyaya Yazıldı.
2021-01-25 17:31:48,549 - INFO - rule_extraction.extraction() is running
2021-01-25 17:32:05,268 - ERROR - url_rules.main() Error : Traceback (most recent call last):
  File "/home/mo/Downloads/Project/PhishDetect/pdd/src/url_rules.py", line 122, in phish_rules_main
    features.update(test)
TypeError: 'bool' object is not iterable

2021-01-25 17:32:23,131 - ERROR - url_rules.main() Error : Traceback (most recent call last):
  File "/home/mo/Downloads/Project/PhishDetect/pdd/src/url_rules.py", line 118, in phish_rules_main
    features.update(test)
TypeError: 'bool' object is not iterable

2021-01-25 17:32:30,515 - ERROR - Error : Traceback (most recent call last):
  File "/home/mo/Downloads/Project/PhishDetect/pdd/src/rule_extraction.py", line 106, in extraction
    flag = self.thread_extraction(line, domain_features)
  File "/home/mo/Downloads/Project/PhishDetect/pdd/src/rule_extraction.py", line 47, in thread_extraction
    driver.get(line['url'])
  File "/home/mo/.local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/mo/.local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 319, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/home/mo/.local/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 374, in execute
    return self._request(command_info[0], url, body=data)
  File "/home/mo/.local/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 397, in _request
    resp = self._conn.request(method, url, body=body, headers=headers)
  File "/usr/lib/python3.9/site-packages/urllib3/request.py", line 78, in request
    return self.request_encode_body(
  File "/usr/lib/python3.9/site-packages/urllib3/request.py", line 170, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/usr/lib/python3.9/site-packages/urllib3/poolmanager.py", line 375, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 445, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 440, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/lib/python3.9/http/client.py", line 1347, in getresponse
    response.begin()
  File "/usr/lib/python3.9/http/client.py", line 307, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.9/http/client.py", line 268, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.9/socket.py", line 704, in readinto
    return self._sock.recv_into(b)
KeyboardInterrupt

2021-01-25 17:32:30,516 - INFO - features Dosyaya Yazıldı.
2021-01-25 17:32:30,517 - INFO - arff Dosyaya Yazıldı.
